Extension/src/LanguageServer/ui.ts

```diff
@@ -238,15 +239,15 @@ export class LanguageStatusUI {
         if (this.isParsingWorkspacePaused) {
             const displayTwoStatus: boolean = this.isParsingFiles && this.isParsingWorkspace;
             return (this.isParsingFiles ? this.parsingFilesTooltip : "")
-                + (displayTwoStatus ? " | " : "")
-                + (this.isParsingWorkspace ? this.workspaceParsingPausedText : "");
+        + (displayTwoStatus ? " | " : "")
+        + (this.isParsingWorkspace ? this.workspaceParsingPausedText : "");
         } else {
             return this.isParsingWorkspace ? this.workspaceParsingRunningText : this.parsingFilesTooltip;
         }
     }
 
     private setTagParseStatus(): void {
-        // Set busy icon outside of timer for more real-time response
+    // Set busy icon outside of timer for more real-time response
```

Comment:
This indentation isn't aligned with the function body?

Reply:
fixed.